spec/models/theme_spec.rb

```diff
@@ -1125,6 +1031,200 @@ def included_settings(id)
     end
   end
 
+  describe "#migrate_settings" do
```

Comment:
I'm not sure if I missed it but it looks like we're missing a test for removing a theme setting